doc/about.rst

```diff
@@ -0,0 +1,47 @@
+.. _about:
+
+======================
+About ``mybinder.org``
+======================
+
+This page explains some of the teams and organizations behind mybinder.org.
+
+What technology runs mybinder.org?
+==================================
+
+The technology behind mybinder.org is primarily composed of three open-source projects:
+
+* `JupyterHub <https://z2jh.jupyter.org>`_, which manages cloud infrastructure for user instances
+* `repo2docker <repo2docker.readthedocs.io>`_, which builds Docker images from GitHub repositories
+* `BinderHub <binderhub.readthedocs.io>`_, which orchestrates the above two projects and
+  provides the Binder interface.
+
+Each of these are open and inclusively-governed projects. Currently, these are all officially
+hosted as a part of Project Jupyter, an open project that creates open tools for data science
+infrastructure and interactive computation. The mybinder.org team is
+heavily involved in each, though mybinder.org does not need to have a controlling
+stake in any of them.
+
+Who runs mybinder.org?
+======================
+
+mybinder.org is a single deployment of the BinderHub technology. It is run as a free, public
+service to the scientific and educational community. It also serves as a way for the Binder
```

Comment:
Maybe not explicitly limit who it is for? For example do we mean to exclude not-for-profits? I don't have a good alternative formulation but maybe something like "... to all those who want to share their work publicly"?